styles/variables-base.less

```diff
@@ -107,6 +109,10 @@
 @agate-button-smallest-badge-letter-spacing: @agate-button-small-badge-letter-spacing;
 @agate-button-smallest-badge-text-indent: @agate-button-small-badge-text-indent;
 @agate-button-smallest-badge-transform: @agate-button-badge-transform;
+@agate-button-text-height: 42px;
+@agate-button-font-stretch: normal;
+@agate-button-font-style: normal;
+@agate-button-text-align: center;
```

Comment:
Let's assign this to the CSS initial value, and then set the skin-specific value in the silicon variables file, so we don't interfere with other skins.

Reply:
I will undo my changes regarding Button component. It was part of another pull request (#253). Thank you!